site/content/blog/kctrl-release-blog.md

```diff
@@ -0,0 +1,56 @@
+
+## What is `kctrl`?
+`kctrl` is kapp-controller's native CLI which help users effectively interact with and observe custom resources exposed by kapp-controller (App, PackageInstall, PackageRepository, Package CRs). The first release of `kctrl` focuses on helping users getting started with consuming packages with minimum friction. In the next few releases, `kctrl` aims to improve observability of resources created using kapp-controller's declarative APIs, improving the debugging experience and helping users authoring Packages and AppCRs to iterate faster and more efficiently.
+
+## How does `kctrl` help package consumers?
+Traditionally, installing a package would involve setting up relevant cluster roles and service accounts which allow the package installations to create desired resources on the cluster. In case the user is configuring the package, they would need to create a Secret which makes this information available and then reference it in their PackageInstall resource. These resources had to be authored and tracked by the user.
+
+`kctrl` helps users get up and running with a pacakage installation with as little as
+```bash
+$ kctrl package install -i cert-man -p cert-manager.community.tanzu.vmware.com --version 1.6.1 --values-file values.yaml
+```
+Where, `values.yaml` defines configuration for the installation.
+
+`kctrl` creates sevice accounts and role bindings with required permissions and keeps track of them. If custom configuration is provided by the user, `kctrl` creates a secret and adds required references to the custom resources being created. When a package is uninstalled, these resources are garbage collected and deleted from the cluster as well.
+
+In case, the user has to use a service account with limited permissions, the same can be supplied to the CLI while installing the application. In this case, kapp-controller does not try to create and track the required resources.
+
+`kctrl` also allows users to add PacakgeRepositories published as `imgpkg` bundles easily to their clusters.
+```bash
+$ kctrl package repository add -r tce --url projects.registry.vmware.com/tce/main:0.10.0
+```
+
+You can find a more detailed end to end workflow for consuming published packages, configuring them and observing them over [here](/kapp-controller/docs/develop/kctrl-package-tutorial/)!
+
+## What is next for `kctrl`?
+We want to help our users to easily observe what kapp-controller is doing. `kctrl` aims to do this by surfacing information about what kapp-controller does with the configuration provided by the user in a transparent, legible and usable manner. 
+
+This enables developers authoring App CRs and Carvel packages to iterate faster by observing how their configuration affects the cluster. It also enables package consumers to narrow down on the root cause of erroneous installations more conveniently.
```

Comment:
nit: The part where we talk about `configuration affecting the cluster` seems a little repetitive to me.

```suggestion
We want to help our users to easily observe what kapp-controller is doing. `kctrl` aims to do this by surfacing information about what kapp-controller does with the configuration provided by the user in a transparent, legible and usable manner. It also enables package consumers to narrow down on the root cause of erroneous installations more conveniently.
```

Reply:
Makes sense, will make this change